scripts/install_third_party_libs.py

```diff
@@ -159,8 +158,8 @@ def compile_protobuf_files(proto_files_paths):
     buf_path = os.path.join(
         BUF_DIR,
         BUF_DARWIN_FILES[0] if common.is_mac_os() else BUF_LINUX_FILES[0])
-    for is_frontend_required, path in proto_files_paths:
-        if is_frontend_required:
+    for generate_js_and_ts_files, path in proto_files_paths:
```

Comment:
"generate" is still a verb.

Try js_and_ts_files_required, perhaps.

Reply:
updated